internal/services/compute/managed_disk_resource_test.go

```diff
@@ -124,6 +124,22 @@ func TestAccManagedDisk_fromGalleryImage(t *testing.T) {
 	})
 }
 
+func TestAccManagedDisk_upload(t *testing.T) {
+	data := acceptance.BuildTestData(t, "azurerm_managed_disk", "test")
+	r := ManagedDiskResource{}
+
+	data.ResourceTest(t, r, []acceptance.TestStep{
+		{
+			Config: r.upload(data),
```

Comment:
Let's also test updating this attribute

Reply:
commented below that update isn't supported by SDK